starter_code/about me.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>About Me</title>
+    </head>
+
+    <body>
+        <link rel="stylesheet" href="about me.css">
+        
+        <div class="topnav">
+            <a href="about me.html">About Me</a>
+            <a href="Portfolio.html">Portfolio</a>
+            <a href="https://www.instagram.com/gavriellag" target="_blank"><img src="images/instagram_white.png" width="20" height="20" alt="instagram"></a>
+            <a href="https://www.behance.net/gavriellagoldrich" target="_blank"><img src="images/behance_white.png" width="20" height="20" alt="behance"></a>
+            <a href="https://www.linkedin.com/in/gavriellagoldrichkelin/" target="_blank"><img src="images/linkedin_white.png" width="20" height="20" alt="linkedin"></a>   
+        </div>
+       
+        <img src= "images/cover photo.jpg" width="1653" height="700">
```

Comment:
@GavriellaGoldrich when you are using names on files and folder make sure there is no white space. For example, "cover photo.jpg" should be "cover-photo.jpg" or use an underscore_